tests/test_docs.py

```
@@ -1,136 +1,99 @@
+            # Extract floats from both strings
+            want_floats = [
+                float(x) for x in want.replace(",", " ").split() if x.replace(".", "", 1).replace("-", "", 1).isdigit()
+            ]
+            got_floats = [
+                float(x) for x in got.replace(",", " ").split() if x.replace(".", "", 1).replace("-", "", 1).isdigit()
+            ]
+
+            if len(want_floats) != len(got_floats):
+                return False
+
+            return all(math.isclose(w, g, rel_tol=1e-3, abs_tol=1e-5) for w, g in zip(want_floats, got_floats))
```

Ruff (0.14.4)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)